src/components/SideMenu/SideMenu.test.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import SideMenu from '.';
+
+describe('SideMenu', () => {
+  beforeEach(() => {
+    render(<SideMenu />);
+  });
+
+  it('renders SideMenu and its elements', () => {
+    const sideMenu = screen.getByTestId('sideMenu');
```

Comment:
Same as other tests, look for the internal elements and their details.  Make your tests meaningful for the readers.  Your tests should explain the usage of your production code